web/modeltr/connection.py

```diff
@@ -42,21 +44,30 @@ def use(cls, alias=DEFAULT_CONNECTION_NAME):
                     alias
                 )
             )
-        return cls.__connections[alias]
+        connection = cls.__connections[alias]
+        try:
+            connection["connection"].client.reset()
+        except (psycopg2.InterfaceError, psycopg2.OperationalError, psycopg2.DatabaseError) as e:
+            connection["connection"] = cls(**connection["args"])
+        return cls.__connections[alias]["connection"]
 
 
+# TODO: delete
```

Comment:
Maybe this TODO could be completed before merging?

Reply:
right